### ISSUE TITLE
BlockArgument support

### ISSUE THREAD

**Issue by paolino** [ennocramer/floskell#32]
Testing BlockArguments syntax

```
-- don't work with and without extension 
f x = identity do
            x

-- don't work with extension
g  x = identity $ do
            x
```

**Comment by ennocramer** [comment]
`BlockArguments` is not yet supported by `haskell-src-exts`. Floskell will not support anything that is not in there.

**Comment by paolino** [comment]
Thanks, don't know if to close it or leave it open for reference. Please decide this for me.

**Comment by domenkozar** [comment]
See https://github.com/haskell-suite/haskell-src-exts/issues/419

**Comment by ennocramer** [comment]
Fixed in `floskell-0.10.3`.
